pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -458,6 +459,11 @@ func (os *OpenStack) ScrubDNS(nameServers, searches []string) ([]string, []strin
 	return nameServers, searches
 }
 
+// KeyManagementService provides a named key management service supported by the cloud.
+func (os *OpenStack) KeyManagementService(name string) (cloudprovider.KeyManagementService, error) {
+	return nil, fmt.Errorf("cloud %q does not support KeyManagementService %q", os.ProviderName(), name)
```

Comment:
I wonder if it would make more sense to return an error saying that provider does not support KeyManagementServices yet. Indicating a name suggests that there is some KMS that they do support.